QUEST_LV_0100.tsv

```diff
@@ -1,124 +1,124 @@
-QUEST_LV_0100_20150317_000093	Statue of Old Owl
-QUEST_LV_0100_20150317_000094	There's no one in this forest. Why are you here?{nl}
-QUEST_LV_0100_20150317_000095	Who? Gaigalas?{nl}Defeating it without any assitance will be difficult.
-QUEST_LV_0100_20150317_000096	I have seen many deaths here.{nl}I don't want to see any more sacrifices.
-QUEST_LV_0100_20150317_000097	I need Kepa Raider juice, and Cronewt leather.
-QUEST_LV_0100_20150317_000098	Why do you have that?{nl}Hand it to me and I will make a Fragment of Memory.
-QUEST_LV_0100_20150317_000099	With this Fragment of Memory I will revive the memories of our sleeping colleagues.
-QUEST_LV_0100_20150317_000100	It will be difficult to defeat it without the Fragment of Memory.{nl}Please bring the horn of Infroholder.
-QUEST_LV_0100_20150317_000101	How come you have all these ingredients?
-QUEST_LV_0100_20150317_000102	
-QUEST_LV_0100_20150317_000103	We can calm them if we defeat Gaigalas.{nl}Please go see the Worrying Owl.
-QUEST_LV_0100_20150317_000104	The Statue of Worrying Owl
-QUEST_LV_0100_20150317_000105	Please wake the sleeping colleagues and help us complete the magic on this day.{nl}With the Fragment of Memory.
-QUEST_LV_0100_20150317_000106	They will be satisfied.{nl}I will stand here while remembering our colleagues.
+QUEST_LV_0100_20150317_000001	$Sentinel
+QUEST_LV_0100_20150317_000002	$If you choose to go to Klaipeda, I recommend going directly through Shadow Forest Road.{nl}The deep woods are dangerous, so try to stay away from there if you can.
+QUEST_LV_0100_20150317_000003	$Man in Hiding
+QUEST_LV_0100_20150317_000004	$Fi... Finally! Has help arrived?{nl}Huh?! it's just a kid? Hey... Are there still Vubbes outside?
+QUEST_LV_0100_20150317_000005	$Ugh. Go away! Don't talk to me! We'll get caught by the Vubbes!{nl}This is my place so leave!
+QUEST_LV_0100_20150317_000006	$What? Don't lie to me! I can smell the Vubbes from here.{nl}You can't fool an adult! Go away! Do not come near this barrel!
+QUEST_LV_0100_20150317_000007	$Fi... Finally! Has help arrived?{nl}What, it's just a kid? What are you looking at? Close the lid and move along!{nl}
+QUEST_LV_0100_20150317_000008	$What's that paper you're holding? It looks familiar...{nl}Let me see that! What are you doing? Hey, there's an adult speaking to you!
+QUEST_LV_0100_20150317_000009	$This paper, it's mine! I'm never giving it away again!!{nl}It's.. not something you need - I'll just rip it apart. There.
+QUEST_LV_0100_20150317_000010	$Sheesh, that's lame...
+QUEST_LV_0100_20150317_000011	$This isn't the first sign to appear after all the Goddesses vanished.{nl}Maybe one of these people will be the one to bring the Goddesses back.
+QUEST_LV_0100_20150317_000012	$Soldier Block
+QUEST_LV_0100_20150317_000013	$Be warned!{nl}Monsters are attacking, so all citizens must evacuate!{nl}But, why are those citizens are in the military zone?{nl}
```

Comment:
Last sentence:
Can't really use "but" like that and you forgot to remove the second "are". I'd rewrite this as "Why are those citizens in the military zone anyway?"